### PR TITLE
Add MDN Docs and Flexbox Froggy

### PR DESCRIPTION
**Pull request by HeyFang**
Best guide and documentation for front end dev... interactive tutorials and guides to learn web development. It offers a structured approach, starting from the basics and progressing to more advanced topics.
Was kinda disappointed to not see it on list, so I took liberaty to add it :>

### CODE REVIEW

**Comment by karlhorky** [comment]
@HeyFang thanks for the suggestion! Good point - not sure why it's not already on the list :)

I was looking at the page linked, and there is a "Complete beginners start here!" section on the left side:

Screenshot 2024-09-15 at 14 47 58

Do you think that it instead belongs in the "Beginner" section instead?

Maybe near the top, eg. below Khan Academy?

**Comment by HeyFang** [comment]
Well I thought the same at first, but compared to other resources in Beginners section... MDN Docs ranges from Basic topics to most advanced ones in complete detail... and its written documentation, so not much interactive either, mostly the web-dev I know won't suggest MDN as starting point for web-dev but as the ideal resource when you are ready to dive deep down.
After all said... its still upto you where wanna add it,am just glad to contribute and see MDN docs in that list :>

**Comment by karlhorky** [comment]
Ok, so you're arguing that MDN is harder to absorb for new learners, even though they mention "complete beginners" on the page? I'm assuming, you mean this because there is a lot of text? Or maybe you have other reasons?

**Comment by HeyFang** [comment]
Oh, I din't know resources get harder as we go down... I just put it randomly in intermidiate section, in that case it would be cool to put mdn docs in lower half of beginner section or top in intermediate.
Tbh i did not give much thought bout ranking, its just that all other resources in beginner section seemed basic compared to mdn... nvm ill fix that

**Comment by HeyFang** [comment]
have a look now :>

**Comment by karlhorky** [comment]
@HeyFang didn't have a chance to look yet :)

I checked now, and position looks good, thanks!

File: README.md [hunk 1]
```diff
@@ -64,6 +69,10 @@ This list aims to be a curated set of high quality educational resources. The av
 **Free** game that teaches the CSS grid system created by [@thomaspark](https://github.com/thomaspark)\
 *(CSS)*
 
+#### [Flexbox Froggy](https://flexboxfroggy.com/)
+**Free** game that teaches the CSS Flexbox in interesting way\
```
Comment (karlhorky) [inline README.md:73]:
```suggestion
**Free** game that teaches the CSS Flexbox\
```

**Comment by HeyFang** [comment]
cool, I hope its ready for merge :>

File: README.md [hunk 2]
```diff
@@ -56,6 +56,11 @@ This list aims to be a curated set of high quality educational resources. The av
 **Free** beginner to intermediate full-stack courses with Ruby and JS learning paths\
 *(HTML, CSS, JavaScript, Ruby, Rails)*
 
+#### [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn)
+**Free** guides and documentation for front-end web development, from basic to advanced concepts.\
+*(HTML, CSS, JavaScript, etc.)*
```
Comment (karlhorky) [inline README.md:61]:
```suggestion
*(HTML, CSS, JavaScript, accessibility, performance, React, Ember, Vue, Svelte, Angular, Git, GitHub)*
```

File: README.md [hunk 3]
```diff
@@ -157,6 +166,7 @@ This list aims to be a curated set of high quality educational resources. The av
 #### [Level Up Tutorials](https://levelup.video/library)
 **Free** video tutorials for web developers and designers\
 *(HTML, CSS, JavaScript, React, Svelte, Vue, Node.js, GraphQL, TypeScript, Deno, GitHub, Figma, Ruby, Drupal, Magento, Wordpress)*
+
```
Comment (karlhorky) [inline README.md:169]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -56,6 +56,11 @@ This list aims to be a curated set of high quality educational resources. The av
 **Free** beginner to intermediate full-stack courses with Ruby and JS learning paths\
 *(HTML, CSS, JavaScript, Ruby, Rails)*
 
+#### [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn)
+**Free** guides and documentation for front-end web development, from basic to advanced concepts.\
+*(HTML, CSS, JavaScript, accessibility, performance, React, Ember, Vue, Svelte, Angular, Git, GitHub)*
+
+
```
Comment (karlhorky) [inline README.md:63]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -56,6 +56,10 @@ This list aims to be a curated set of high quality educational resources. The av
 **Free** beginner to intermediate full-stack courses with Ruby and JS learning paths\
 *(HTML, CSS, JavaScript, Ruby, Rails)*
 
+#### [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn)
+**Free** guides and documentation for front-end web development, from basic to advanced concepts.\
```
Comment (karlhorky) [inline README.md:60]:
```suggestion
**Free** guides on fundamental web development concepts\
```

File: README.md [hunk 6]
```diff
@@ -56,6 +56,10 @@ This list aims to be a curated set of high quality educational resources. The av
 **Free** beginner to intermediate full-stack courses with Ruby and JS learning paths\
 *(HTML, CSS, JavaScript, Ruby, Rails)*
 
+#### [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn)
```
Comment (karlhorky) [inline README.md:59]:
```suggestion
#### [MDN Learning Area](https://developer.mozilla.org/en-US/docs/Learn)
```

**Review by karlhorky — APPROVED** [review]

**Pull request merged**